templates/rag-fusion/README.md

```diff
@@ -1,5 +1,66 @@
-# RAG Fusion
+# rag-fusion
 
-Re-implemented from [this GitHub repo](https://github.com/Raudaschl/rag-fusion), all credit to original author
+RAG Fusion is a re-implementation of the project found at [this GitHub repo](https://github.com/Raudaschl/rag-fusion). All credit goes to the original author. It's a search methodology that aims to bridge the gap between traditional search paradigms and the multifaceted dimensions of human queries. Inspired by the capabilities of Retrieval Augmented Generation (RAG), this project goes a step further by employing multiple query generation and Reciprocal Rank Fusion to re-rank search results.
```

Comment:
do we want the indented quoteblock back?

Reply:
cleaned this.